src/PriceIndicator/PriceIndicator.stories.mdx

```diff
@@ -54,7 +54,7 @@ PriceIndicator is a simple styled and animated set of elements to be use to indi
           <button onClick={() => updatePrice({ priceIncrease: Math.random() })}>Increase price by random penny's</button>
           <div style={{ position: 'absolute', bottom: '24px', right: '24px' }}>
             <PriceIndicator isActive total={priceTotal} addition={priceIncrease}>
-              <Tooltip content={<div>
+              <Tooltip trigger="click" offset={[0, 16]} placement="top-start" interactive={true} interactiveBorder={30} content={<div>
```

Comment:
How come this has trigger etc now?

Reply:
This is needed to get it closer to what will be needed n the maps app. I Talked to Sylvia about something yesterday and she mentioned the tooltip will work with a click rather than a hover.